examples/notebooks/launchables/GPU_Cluster_Sizing_with_NeMo_Agent_Toolkit.ipynb

```
@@ -0,0 +1,1166 @@
+    "os.environ[\"PATH\"]=f\"/home/ubuntu/.local/bin:{os.environ['PATH']}\"\n",
+    "\n",
+    "# Install Git LFS\n",
+    "!sudo apt-get install git-lfs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup: Clone the NeMo Agent Toolkit Repository\n",
+    "\n",
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)